webapp/views.py

```diff
@@ -70,7 +77,7 @@ def search(request):
             context["results"] = _get_search_results(query, start, num, site)
 
             if "searchInformation" in context["results"]:
-                context["estimatedTotal"] = int(
+                context["estimatedTotal"] = str_to_int(
```

Comment:
How could this ever not be a number?

Reply:
yeah here it shouldn't be an issue. I mostly did this for all of castings. It can be removed